### PR TITLE
[#801] Export Metadata Subfield Formats

### PR DESCRIPTION
**Pull request by Meowcenary**
This should be merged after 791 and 792 since it rebases off of those. Very small and simple PR.

### CODE REVIEW

**Comment by Meowcenary** [comment]
Below are all the values from the grants_and_funding field on production DigitalHub. There's definitely a way to programmatically find the values we need for some of these, but we'll need to do another mapping similar to what was done in #812 with the known_organizations module.. 

```
"F37 LM009568 NIH",
"F37 LM009568 NIH",
"K01LM011973-01 (NLM)",
"Clinical and Translational Science Award (CTSA) award UL1RR025741",
"Supported by grants from the Swiss National Foundation for Scientific Research (3200B0-108261) and the Mach-Gaensslen Foundation, Switzerland.",
"These studies were supported by United States Public Health Services grants HE 07057 and HE 10573, by the Otho S. A. Sprague Foundation, and by the Ernest S. Bazley Trust.",
"This research was supported in part by an appointment to the NLM Associate Fellowship Program sponsored by the National Library of Medicine and administered by the Oak Ridge Institute for Science and Education. ",
"Justin Smith was supported by National Center for Chronic Disease Prevention and Health Promotion of the Centers of Disease Control and Prevention grant DP006255, awarded to Justin Smith and Cady Berkel, and National Institute on Drug Abuse grant DA027828, awarded to C. Hendricks Brown. Zorash Montao was supported by minority fellowship SM60563-40 from the Department of Health and Human Services. Seed funding from the College of Liberal Arts and Sciences at Arizona State University awarded to Thomas Dishion supported Justin Smith and Zorash Montao.",
"2P30DA027828-07",
"K01LM011973-01 (NLM)",
"Clinical and Translational Science Award (CTSA) award UL1TR001422",
"UL1TR001422",
"This report was made possible with help from the Third Coast Center for AIDS Research (CFAR), an NIH funded center (P30 AI117943), including funding from an administrative supplement to that grant.",
"NCATS grant U24TR002306 (CD2H) ",
"NCATS grant UL1TR001422 (NUCATS)",
"We gratefully acknowledge support for this work by the National Institutes of Health's National Center for Advancing Translational Sciences, Grant Number UL1TR001422",
"UL1TR001422 ",
"This work was supported by the Center for Reproductive Health After Disease (P50HD076188) from the National Institutes of Health National Center for Translational Research in Reproduction and Infertility (NCTRI).",
"This work was supported by the Center for Reproductive Health After Disease (P50HD076188) from the National Institutes of Health National Center for Translational Research in Reproduction and Infertility (NCTRI).",
"This work was supported by the Center for Reproductive Health After Disease (P50HD076188) from the National Institutes of Health National Center for Translational Research in Reproduction and Infertility (NCTRI).",
"This work was supported by the Center for Reproductive Health After Disease (P50HD076188) from the National Institutes of Health National Center for Translational Research in Reproduction and Infertility (NCTRI).",
"This work was supported by the Center for Reproductive Health After Disease (P50HD076188) from the National Institutes of Health National Center for Translational Research in Reproduction and Infertility (NCTRI).",
"This work was supported by the Center for Reproductive Health After Disease (P50HD076188) from the National Institutes of Health National Center for Translational Research in Reproduction and Infertility (NCTRI).",
"This work was supported by the Center for Reproductive Health After Disease (P50HD076188) from the National Institutes of Health National Center for Translational Research in Reproduction and Infertility (NCTRI).",
"U24TR002306",
"UL1TR001422 (NCATS), U54CA202995, U54CA202997, and U54CA203000 (NCI), G08LM012688 (NLM), U24TR002306 (NCATS)",
"U24TR002306 (NCATS)",
"UL1TR001422 (NCATS)",
"U54CA202995 (NCI)",
"U54CA202997 (NCI)",
"U54CA203000 (NCI)",
"P30AR072579 (NIAMS)",
"G08LM012688 (NLM)",
"NCATS U24TR002306",
"NCATS UL1TR001422",
"NIH: U24TR002306 (NCATS)",
"NIH: UL1TR001422 (NCATS)",
"NIH: U54CA202995, U54CA202997, U54CA203000 (NCI)",
"NIH: P30AR072579 (NIAMS)",
"NIH: G08LM012688 (NLM)",
"U24TR002306",
"UL1TR001",
"HBNU Fogarty Global Health Fellows Program; Northwestern University Stamler Fellowship",
"Research reported in this publication was supported by the Fogarty International Center of the National Institutes of Health under Award Number D43TW009575 (RLM and LH) and U54CA221205 (LH and RLM), AASLD Clinical and Translational Research Award in Liver Diseases (RAH), and R01DK110024 and R01AA027179 (KDR). The content is solely the responsibility of the authors and does not necessarily represent the official views of the National Institutes of Health.",
"CTS Personas is a project of the CTSA Program National Center for Data to Health, Grant U24TR002306",
"IGH ",
"See content of this document: PRO Planning Toolkit Overview",
"See PRO Planning Tools Overview",
"See PRO Planning Tools Overview",
"See PRO Planning Tools Overview",
"NIH/NHLBI R01HL115828",
"NIH/NHLBI R01HL133504",
"NIH/NHLBI K25HL119608",
"NIH/NHLBI F30HL145995",
"NIH - National Institute of Diabetes and Digestive and Kidney Diseases",
"This project is funded in part with Federal funds from the National Institute on Aging under grant number U2CAG057441, and The Environmental Influences on Child Health Outcomes (ECHO) program, Office of the Director, NIH, under award number U24OD023319."
```

**Comment by Meowcenary** [comment]
An example of some of the supported identifiers: Screen Shot 2021-01-19 at 3 12 14 PM

**Comment by Meowcenary** [comment]
Supported schemes with links to how the schemes are structured
- [CrossRef Funder ID](https://www.crossref.org/education/funder-registry/funding-data-overview/)
- [GRID](https://www.grid.ac/format)
- [ISNI](https://en.wikipedia.org/wiki/International_Standard_Name_Identifier)
- ROR
- Wikidata

**Comment by Meowcenary** [comment]
Well, I was more tired than I thought today and all of these comments are on a PR and not the ticket I thought I was entering them into. Yikes. I'm going to consolidate and move these comments over now.

**Comment by Meowcenary** [comment]
I'm going to rebase this now that we have merged some more work.

File: app/models/invenio_rdm_record_converter.rb [hunk 1]
```diff
@@ -137,9 +138,4 @@ def creators(creators)
       }
     end
   end
-
-  def format_creator_names(creator_name)
-    family_name = creator_name.pop # remove last value from display name
-    given_name = creator_name.join(' ') # the remaining strings becomes given name
-  end
```
Comment (sharpattack) [inline app/models/invenio_rdm_record_converter.rb:144]: Hah, I didn't catch this earlier either.

**Review by sharpattack — APPROVED** [review]
Another straightforward change 👍 Thanks.

**Review by carsonicator — COMMENTED** [review]
Thanks @Meowcenary

**Pull request merged**